src/components/Avatar/Avatar.tsx

```diff
@@ -59,10 +64,11 @@ export default class Avatar extends React.Component<AvatarProps, {}> {
   static defaultProps: Partial<AvatarProps> = {
     borderType: AvatarBorderType.ROUND,
     size: AvatarSize.MEDIUM,
+    imageShouldFadeIn: true,
```

Comment:
Should it default to `true`? I think designers were complaining about the fade-in behavior, so I'd default it to `false`.

Reply:
ok, sure.

Reply:
updated